RDMMonitor.js

```diff
@@ -558,11 +551,143 @@ function SendOfflineDeviceDMs()
 
     setTimeout(SendOfflineDeviceDMs,60000);
     return;
-    
+}
+
+function ReopenWarnGame()
+{
+    if(!config.allowReopenGame)
+    {
+        return;
+    }
+    else
+    {
+        let now = new Date();
+        now = now.getTime();
+
+        let reopenDevices = [];
+
+        for(var deviceName in devices)
+        {
+            let device = devices[deviceName];
+            let lastSeen = new Date(0);
+            lastSeen.setUTCSeconds(device.lastSeen);
+            lastSeen = lastSeen.getTime();
+            lastSeen = now - lastSeen;
+            if(lastSeen > reopenTime)
+            {
+                if(!config.excludeFromReopen.includes(deviceName))
+                {
+                    reopenDevices.push(device.name);
+                }
+            }
+        }
+
+        for(var i = 0; i < reopenDevices.length; i++)
+        {
+            if(!devices[reopenDevices[i]].reopened)
+            {
+                for(var ii = 0; ii < config.reopenMonitorURL.length; ii++) {
+                    const options = {
+                        url: config.reopenMonitorURL[ii],
+                        json: true,
+                        method: 'POST',
+                        body: { 'type': 'reopen', 'device': devices[reopenDevices[i]].name },
+                        headers: { 'Accept': 'application/json', 'Content-Type': 'application/json' },
+                    };
+                    console.info(GetTimestamp()+`Sending reopen game request for ${devices[reopenDevices[i]].name} to remote listener ${config.reopenMonitorURL[ii]}`);
+                    request(options, (err, res, body) => {
+                        if (err) {
+                            console.error(GetTimestamp()+`Failed to send reopen game request to remote listener for ${options.body.device}`);
+                        }
+                    });
+                    devices[reopenDevices[i]].reopened = true;
+                }
+            }
+        }
+
+        for(var deviceName in devices)
+        {
+            if(devices[deviceName].reopened && reopenDevices.indexOf(deviceName) == -1)
+            {
+                devices[deviceName].reopened = false;
+                console.info(GetTimestamp()+`Device ${devices[deviceName].name} has come back online from reopening the game`);
+            }
+        }
+
+        setTimeout(ReopenWarnGame,60000);
+        return;
+    }
+}
+
+function RebootWarnDevice()
+{
+    if(!config.allowWarnReboots)
+    {
+        return;
+    }
+    else
+    {

```

Comment:
```suggestion
    if(!config.allowWarnReboots)
    {
        return;
    }
```
Remove unnecessary `else` statement